index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>animalcules</title>
+        <link href="canvas.css"rel="stylesheet" type="text/css">
+        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
```

Comment:
Please remove the jQuery dependency, your 95% source code are not using this library.